.env.example

```
@@ -15,6 +15,16 @@ SERVER_MAX_REQUEST_BODY_BYTES=20971520
 SERVER_TLS_CERTFILE=cert.pem
 SERVER_TLS_KEYFILE=key.pem
 SERVER_ENABLE_TLS=true
+GRPC_HOST=0.0.0.0
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The GRPC_HOST key should go before the SERVER_ALLOWED_ORIGINS key

(UnorderedKey)